// puppeteer-core/lib/cjs/puppeteer/puppeteer-core.js
"use strict";
/**
 * Copyright 2017 Google Inc. All rights reserved.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
Object.defineProperty(exports, "__esModule", { value: true });
exports.unregisterCustomQueryHandler = exports.registerCustomQueryHandler = exports.networkConditions = exports.launch = exports.executablePath = exports.errors = exports.devices = exports.defaultArgs = exports.customQueryHandlerNames = exports.createBrowserFetcher = exports.connect = exports.clearCustomQueryHandlers = void 0;
const initializePuppeteer_js_1 = require("./initializePuppeteer.js");
const puppeteer = (0, initializePuppeteer_js_1.initializePuppeteer)('puppeteer-core');
exports.clearCustomQueryHandlers = puppeteer.clearCustomQueryHandlers, exports.connect = puppeteer.connect, exports.createBrowserFetcher = puppeteer.createBrowserFetcher, exports.customQueryHandlerNames = puppeteer.customQueryHandlerNames, exports.defaultArgs = puppeteer.defaultArgs, exports.devices = puppeteer.devices, exports.errors = puppeteer.errors, exports.executablePath = puppeteer.executablePath, exports.launch = puppeteer.launch, exports.networkConditions = puppeteer.networkConditions, exports.registerCustomQueryHandler = puppeteer.registerCustomQueryHandler, exports.unregisterCustomQueryHandler = puppeteer.unregisterCustomQueryHandler;
exports.default = puppeteer;
